### PR TITLE
[DSS] CMI: Update OS Email Invite to Sign Germline Consent

### PR DESCRIPTION
**Pull request by aweng98**
[PEPPER-484](https://broadworkbench.atlassian.net/browse/PEPPER-484)

The automated OS email sent to participants upon tissue AND blood or saliva receipt to invite them to sign the germline consent is updated per Acceptance Criteria.

Deployment instr is in Jira ticket.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [ ] I have assessed potential risks and labeled using the `R-*` labels.
- [ ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ ] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have written zero automated tests but have poked around locally to verify proper functionality
- [x] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

[PEPPER-484]: https://broadworkbench.atlassian.net/browse/PEPPER-484?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/osteo/emails/parental-somatic-germline-addendum-required.html [hunk 1]
```diff
@@ -83,7 +83,7 @@
 
     <tr style="width: 540px;">
         <td style="padding: 50px 0 0 0;">
-            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear -ddp.proxy.firstName-,</p>
+            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear Parent,</p>
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/emails/parental-somatic-germline-addendum-required.html]: Parent name should be substituted in the email. I believe existing version is correct and no change needed here

File: pepper-apis/studybuilder-cli/studies/osteo/emails/self-somatic-germline-addendum-required.html [hunk 2]
```diff
@@ -83,7 +83,7 @@
 
     <tr style="width: 540px;">
         <td style="padding: 50px 0 0 0;">
-            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear -ddp.participant.firstName-,</p>
+            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear Participant,</p>
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/emails/self-somatic-germline-addendum-required.html]: Participant name should be substituted in the email. I believe existing version is correct and no change needed here

**Review by ssettipalli — CHANGES_REQUESTED** [review]

File: pepper-apis/studybuilder-cli/studies/osteo/emails/self-somatic-germline-addendum-required.html (hunk 2 above)
Comment (aweng98) [reply pepper-apis/studybuilder-cli/studies/osteo/emails/self-somatic-germline-addendum-required.html]: Good catch. Reverted greeting.

**Review by ssettipalli — APPROVED** [review]
looks good to me

**Pull request merged**